### ISSUE TITLE
Converting tiny-yolov3 gives input/output error

### ISSUE THREAD

**Issue by SlavaKeshkov** [jinyu121/DW2TF#32]
Hello @jinyu121. I am trying out the new release that you made for tiny-yolov3 but facing errors during the conversion. I used your code to convert the models:

'python main.py \
    --cfg 'data/yolov3-tiny.cfg' \
    --weights 'data/yolov3-tiny.weights' \
    --output 'data/' \
    --prefix 'yolov3-tiny/' \
    --gpu 0'

I used original darkflow weights file and the cfg file. However I am getting the error below.

```
UnknownError (see above for traceback): Failed to rename: data/yolov3-tiny.ckpt.index.tempstate5997668412469460368 to: data/yolov3-tiny.ckpt.index : Access is denied.
; Input/output error

```
![error](https://user-images.githubusercontent.com/44873863/62608451-239ad900-b900-11e9-9e0a-5c9b3c12dcf6.PNG)

As output I get the following files (see screenshot). I am uncertain if the conversion has been completed. Please let me know!

![output](https://user-images.githubusercontent.com/44873863/62608282-e0406a80-b8ff-11e9-9b48-46e3f1dba90a.PNG)

**Comment by albayrakonur** [comment]
did you solve the problem ? I have same one

**Comment by robinydv** [comment]
@albayrakonur Any update? I am also experiencing the same problem.

**Comment by SlavaKeshkov** [comment]
I switched to another repo for conversion, this one:
https://github.com/zzh8829/yolov3-tf2

**Comment by robinydv** [comment]
@SlavaKeshkov Hi, did you train on darknet with a custom dataset? I also used that repo for conversion(custom dataset with 1 class) but I get worse results.

**Comment by SlavaKeshkov** [comment]
Yes trained on a custom one. Not sure why there is a difference. I had many
classes though, not one. Also look into the threshold value when you
convert, it could be causing the issue

**Comment by robinydv** [comment]
@SlavaKeshkov Thanks for the quick reply. Are you talking about the IoU threshold or the confidence threshold?

**Comment by SlavaKeshkov** [comment]
I believe the confidence one
